apps/frontend/src/libs/components/icon/libs/maps/icon-name-to-svg.map.ts

```diff
@@ -6,6 +6,7 @@ import EyeIcon from "~/assets/icons/eye.svg?react";
 import EyeOffIcon from "~/assets/icons/eye-off.svg?react";
 import HomeIcon from "~/assets/icons/home.svg?react";
 import LogOutIcon from "~/assets/icons/log-out.svg?react";
+import NotificationBell from "~/assets/icons/notification-bell.svg?react";
```

Comment:
```suggestion
import NotificationBellIcon from "~/assets/icons/notification-bell.svg?react";
```

And maybe you can simplify it from NotificationBell to Bell